physionet-django/console/templates/console/expired_training_table.html

```diff
@@ -17,6 +18,13 @@
                     <td>{{ training.user.get_full_name }}</td>
                     <td>{{ training.user.email }}</td>
                     <td>{{ training.training_type.name }}</td>
+                    <td>
+                        {% if training.user.is_credentialed %}
```

Comment:
This can be simplified with the Django "yesno" template tag (see: https://docs.djangoproject.com/en/4.1/ref/templates/builtins/#yesno).

e.g. I think you should be able to replace the if clause with something like this: 
```
{{ training.user.is_credentialed|yesno:"Yes,No" }}
```

Reply:
@tompollard I did the suggested changes, Tom. Thank you so much.